README.md

```diff
@@ -134,8 +134,8 @@ const { webkit } = require('playwright');
 
 ## Resources
 
-* [Documentation](docs/README.md)
-* [API reference](docs/api.md)
-* [Example recipes](docs/examples/README.md)
+* [Documentation](https://playwright.dev/)
```

Comment:
This should be `https://playwright.dev/#?path=docs/README.md`

Reply:
Done